lib/ansible/modules/network/fortimanager/fmgr_fwpol_package.py

```diff
@@ -31,6 +31,18 @@
 version_added: "2.8"
 notes:
     - Full Documentation at U(https://ftnt-ansible-docs.readthedocs.io/en/latest/).
+    - Revision Comments April 2nd 2019
```

Comment:
@ftntcorecse Revision comments should be added to backports in the form of changelogs. You can refer to https://github.com/ansible/ansible/pull/56399/files.

More on backport process: https://docs.ansible.com/ansible/latest/community/development_process.html#backporting-merged-prs